plugins/dbviewer/api/api.js

```diff
@@ -275,6 +294,63 @@ var common = require('../../../api/utils/common.js'),
                     });
                 }
             }
+            else if ((params.qstring.dbs || params.qstring.db) && params.qstring.collection && params.qstring.collection.indexOf('system.indexes') === -1 && params.qstring.collection.indexOf('sessions_') === -1 && params.qstring.aggregate) {
+                var match, group = {};
+                if (params.member.global_admin) {
+                    if (params.qstring.group) {
+                        try {
+                            if (params.qstring.match) {
+                                match = JSON.parse(params.qstring.match) || {};
```

Comment:
match will never be assigned the {}, if incase JSON.parse fails.
I think you should initialize match = {}, just like done with group.